tests/test_unitxt.py

```diff
@@ -0,0 +1,24 @@
+# First Party
```

Comment:
Note that with:

https://github.com/instructlab/eval/pull/165

This file should move to scripts/